Merge update

FlyingShuriken opened the pull request.

transitive-bullshit committed:
bump deps

transitive-bullshit committed:
feat: update minimum node.js requirement

transitive-bullshit committed:
bump deps

dsarman committed:
fix: add header background for dark mode

transitive-bullshit committed:
bump deps

transitive-bullshit committed:
🎳

transitive-bullshit committed:
👬

transitive-bullshit committed:
bump deps; replace collection dropdown

transitive-bullshit committed:
bump deps; optimize collection and next/image re-renders

dsarman committed:
Add dark mode header background for Firefox only

fix: Added dark mode header background for Firefox to work around it not supporting backdrop-filter

Vadorequest committed:
Add PostHog analytics

transitive-bullshit committed:
😴

transitive-bullshit committed:
bump deps

transitive-bullshit committed:
👜

transitive-bullshit committed:
feat: add caching for page URI to notion IDs

Vadorequest committed:
Remove comments

Vadorequest committed:
Use ^ version

Vadorequest committed:
Merge branch 'main' into add-posthog-analytics

# Conflicts:
#	yarn.lock

Vadorequest committed:
Update yarn.lock (fix conflict)

transitive-bullshit committed:
Merge pull request #254 from Vadorequest/add-posthog-analytics

Vadorequest committed:
Add NOTE: don't forget to set isRedisEnabled to true in the site.config.js file

Vadorequest committed:
Add Redis documentation

transitive-bullshit committed:
Merge pull request #255 from Vadorequest/redis-doc

transitive-bullshit committed:
feat: add support for social images

transitive-bullshit committed:
feat: switch react-icons import style

transitive-bullshit committed:
Merge pull request #256 from transitive-bullshit/feature/social-preview-images

transitive-bullshit committed:
🔃

transitive-bullshit committed:
🎹

transitive-bullshit committed:
Merge pull request #251 from dsarman/fix_dark_mode_header_background

transitive-bullshit committed:
🛢

transitive-bullshit committed:
Merge branch 'main' of github.com:transitive-bullshit/nextjs-notion-starter-kit into main

* 'main' of github.com:transitive-bullshit/nextjs-notion-starter-kit:
  Add dark mode header background for Firefox only
  fix: add header background for dark mode

transitive-bullshit committed:
🛋

transitive-bullshit committed:
feat: change social image urls

transitive-bullshit committed:
fix: social images

transitive-bullshit committed:
🗻

transitive-bullshit committed:
👻

transitive-bullshit committed:
🚜

transitive-bullshit committed:
👇

transitive-bullshit committed:
👨

transitive-bullshit committed:
💯

transitive-bullshit committed:
😗

transitive-bullshit committed:
🔱

transitive-bullshit committed:
😿

transitive-bullshit committed:
👘

transitive-bullshit committed:
👁

transitive-bullshit committed:
🛣

transitive-bullshit committed:
🍟

transitive-bullshit committed:
bump deps

transitive-bullshit committed:
feat: minor font fixes

transitive-bullshit committed:
feat: clean up default favicons

transitive-bullshit committed:
⏸

transitive-bullshit committed:
🍱

transitive-bullshit committed:
bump deps

transitive-bullshit committed:
👣

transitive-bullshit committed:
🦋

transitive-bullshit committed:
♓

transitive-bullshit committed:
👆

transitive-bullshit committed:
feat: WIP custom header nav

transitive-bullshit committed:
👅

transitive-bullshit committed:
⤵

transitive-bullshit committed:
🖲

transitive-bullshit committed:
🍘

transitive-bullshit committed:
Merge pull request #259 from transitive-bullshit/feature/custom-navigation

transitive-bullshit committed:
🚨

transitive-bullshit committed:
🌓

transitive-bullshit committed:
🍮

transitive-bullshit committed:
bump deps

transitive-bullshit committed:
feat: pin chrome-aws-lambda to 2.1.1

transitive-bullshit committed:
feat: pin chrome-aws-lambda to 5.5.0

transitive-bullshit committed:
🌷

transitive-bullshit committed:
🔻

transitive-bullshit committed:
🖊

transitive-bullshit committed:
feat: improve page metadata display

transitive-bullshit committed:
🎩

transitive-bullshit committed:
🥖

transitive-bullshit committed:
🈯

transitive-bullshit committed:
🐯

transitive-bullshit committed:
⛩

transitive-bullshit committed:
feat: bump deps; syntaxes now optional; katex now peer dep

transitive-bullshit committed:
📵

transitive-bullshit committed:
🔻

transitive-bullshit committed:
remove CustomFont as its not being used

transitive-bullshit committed:
feat: spice up links a bit

transitive-bullshit committed:
feat: update third-party deps

transitive-bullshit committed:
chore: refactor react import

transitive-bullshit committed:
bump deps

transitive-bullshit committed:
bump deps

transitive-bullshit committed:
bump deps

transitive-bullshit committed:
feat: add more logging to getSiteMaps

transitive-bullshit committed:
remove katex as peer dep

transitive-bullshit committed:
🌵

transitive-bullshit committed:
♒

transitive-bullshit committed:
feat: replace use-dark-mode with next-themes

transitive-bullshit committed:
🐆

transitive-bullshit committed:
feat: revert to using official twitter embedding sdk; bump deps

transitive-bullshit committed:
bump deps

transitive-bullshit committed:
fix: notion link hover+focus underline animation

transitive-bullshit committed:
🐀

transitive-bullshit committed:
feat: fix dark mode; minor optimizations

transitive-bullshit committed:
feat: add rss feed and cleanup some sitemap code

transitive-bullshit committed:
Merge pull request #273 from transitive-bullshit/feature/rss

transitive-bullshit committed:
feat: update notion quote styles

transitive-bullshit committed:
🥜

transitive-bullshit committed:
🚉

transitive-bullshit committed:
🎺

transitive-bullshit committed:
🚧

transitive-bullshit committed:
📗

transitive-bullshit committed:
🏨

transitive-bullshit committed:
⏲

transitive-bullshit committed:
😞

transitive-bullshit committed:
🌗

transitive-bullshit committed:
📎

transitive-bullshit committed:
feat: revert dark mode to @fisch0920/use-dark-mode

transitive-bullshit committed:
🎥

transitive-bullshit committed:
🗄

si1k committed:
Adding YouTube social icon

si1k committed:
Fixed whitespace differences

si1k committed:
Adding newsletter icon

si1k committed:
Commenting youtube from default config

si1k committed:
Commenting newsletter in default config

transitive-bullshit committed:
Merge pull request #315 from si1k/youtube-icon

transitive-bullshit committed:
Merge branch 'main' into newsletter-icon

transitive-bullshit committed:
Merge pull request #317 from si1k/newsletter-icon

transitive-bullshit committed:
👀

transitive-bullshit committed:
📼

si1k committed:
Adding meta tags to remove bars on safari browser

si1k committed:
Updating bar style to black

bytrangle committed:
feat(types): add interface for Params

bytrangle committed:
feat: add type safety for getStaticProps

transitive-bullshit committed:
Merge pull request #330 from bytrangle/getStaticProps-type-checking

feat: add type safety for getStaticProps

Commit:
update next to 12.2.3

transitive-bullshit committed:
Merge pull request #353 from dev0x56/feature/update-deps

JoKenPo committed:
fix: dark mode color when loading page

GridexX committed:
fix: dark mode for searchInput

transitive-bullshit committed:
feat: update deps

transitive-bullshit committed:
feat: update github actions

transitive-bullshit committed:
feat: switch OG images to vercel edge runtime + Satori

transitive-bullshit committed:
🚯

transitive-bullshit committed:
⛪

transitive-bullshit committed:
🈲

transitive-bullshit committed:
🗂

transitive-bullshit committed:
💐

transitive-bullshit committed:
Merge pull request #383 from transitive-bullshit/feature/update-deps

transitive-bullshit committed:
Merge pull request #379 from r2devops/main

transitive-bullshit committed:
🕯

transitive-bullshit committed:
🗜

transitive-bullshit committed:
👸

transitive-bullshit committed:
🛐

transitive-bullshit committed:
chore: prettier-plugin-sort-imports

transitive-bullshit committed:
🔞

transitive-bullshit committed:
🖼

transitive-bullshit committed:
Merge pull request #386 from transitive-bullshit/feature/prettier-plugin-sort-imports

transitive-bullshit committed:
feat: use ky instead of fetch for edge func

transitive-bullshit committed:
🏌

transitive-bullshit committed:
Merge pull request #387 from transitive-bullshit/feature/ky

transitive-bullshit committed:
👭

transitive-bullshit committed:
🕴

transitive-bullshit committed:
feat: remove ky in favor of fetch

transitive-bullshit committed:
fix: add presence test for OG image assets

voidrender committed:
✨ Add support for Mastodon link in footer

transitive-bullshit committed:
Merge pull request #402 from ioveracker/mastodon

Vadorequest committed:
Update posthog-js + Fix PostHog type

transitive-bullshit committed:
Merge pull request #416 from Vadorequest/fix-posthog-types

transitive-bullshit committed:
chore: update year to 2023

kimwonj77 committed:
filter non-public page on sitemap

engageintellect committed:
Make header scrollable

Longer page titles text to overflow off the right side of the screen. This small change will enable users to scroll the header horizontally on mobile devices so they can read the whole page title

ohplz committed:
fix: remove the transparent line above the header

Commit:
feature: optmize scrollbar style

transitive-bullshit committed:
Merge pull request #469 from ohplz/fix/remove_transparent_line

transitive-bullshit committed:
Merge pull request #470 from ohplz/feature/optmize_scrollbar_style

Commit:
feature: optimize header's backdrop-filter in dark mode.

Commit:
feature: set color for mobile browser status bar

it'll diffirent due to the dark or light mode of mobile devices.

transitive-bullshit committed:
Merge pull request #475 from ohplz/feature/browser_status_bar_theme_color

hongzzz committed:
feature: Update copyright year automatically

kaichii committed:
fix search dialog for dark mode

Commit:
feat(footer): get copyright year from current date

Commit:
fix: Problem that Code block is not displayed if even one import fails.

jonleung committed:
Update instructions for how to use navigationLinks

I was confused by this for ~2 hours. I didn't realize that navigationStyle: 'custom' needed to be enabled to use the navigationLinks and was digging through the code wondering why navigationLinks were not showing.

jonleung committed:
Merge pull request #1 from byjonathanleung/clarify-site-config-comments-for-navigation-links

Update instructions for how to use navigationLinks

transitive-bullshit committed:
Merge pull request #506 from mosugi/fix-code-not-displayed

omarcostahamido committed:
Update package.json

- update `react-use` to avoid conflict while installing

omarcostahamido committed:
center equation blocks

transitive-bullshit committed:
Merge pull request #545 from omarcostahamido/patch-3

transitive-bullshit committed:
Merge pull request #541 from omarcostahamido/patch-1

transitive-bullshit committed:
feat: update deps

transitive-bullshit committed:
Merge pull request #515 from byjonathanleung/main

transitive-bullshit committed:
Merge pull request #459 from kimwonj77/sitemap-filter

transitive-bullshit committed:
Merge pull request #505 from anqfelix/feat/enhance-footer

transitive-bullshit committed:
Merge branch 'main' into fix-dark-mode-search-dialog

transitive-bullshit committed:
Merge pull request #481 from kaichii/fix-dark-mode-search-dialog

transitive-bullshit committed:
Merge branch 'main' into feature/copyright

transitive-bullshit committed:
Merge pull request #479 from hongzzz/feature/copyright

transitive-bullshit committed:
Merge pull request #474 from ohplz/feature/optimize_dark_header_blur

transitive-bullshit committed:
Merge pull request #464 from engageintellect/patch-1

transitive-bullshit committed:
Merge pull request #368 from JoKenPo/main

transitive-bullshit committed:
Merge pull request #323 from si1k/remove-mobile-whitespace

onnimonni committed:
Document the Deployment Protection setting causing 401 Unauthorized for preview images in Vercel. fixes #435, fixes #437

ghost commented:
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
LevanisART, @radonn24, @dulumao, @devArno88, @wisdomabioye, @MarshallChang, @onepisYa, @py7hon, @4pmtong, @chapter802, @real-lazy-coder, @haythemsellami, @emg110, @AdemKao, @agnescameron, @Nach0Git, @lyknice23, @0xYakuza, @lichenysq, @gHashTag

transitive-bullshit committed:
fix: locally linked

transitive-bullshit committed:
🐕

transitive-bullshit committed:
feat: update react-notion-x to v7

transitive-bullshit committed:
fix

transitive-bullshit committed:
🦌

transitive-bullshit committed:
Merge pull request #641 from transitive-bullshit/feature/maintenance-fall-2024

transitive-bullshit committed:
🈂

transitive-bullshit committed:
fix: social images

transitive-bullshit committed:
fix: update react-notion-x

transitive-bullshit committed:
fix: fonts issue

transitive-bullshit committed:
fix

transitive-bullshit committed:
Merge pull request #651 from transitive-bullshit/fix/social-images

transitive-bullshit committed:
chore: update lqip-modern

transitive-bullshit committed:
chore: update lqip-modern

onnimonni committed:
Replace Vercel Deployment Protection setting screenshot with short GIF

transitive-bullshit committed:
Merge pull request #594 from onnimonni/document-fix-for-social-media-preview-images

transitive-bullshit committed:
🍓

transitive-bullshit committed:
feat: remove inter regular font and just use bold for social images

transitive-bullshit committed:
feat: improve tweet embedding using react-tweet

transitive-bullshit committed:
♀

onnimonni committed:
Fix apple-mobile-web-app-capable deprecated meta tag warning

transitive-bullshit committed:
Merge pull request #655 from onnimonni/change-deprecated-mobile-web-meta-tag

Fix apple-mobile-web-app-capable deprecated meta tag deprecated warning

transitive-bullshit committed:
feat: update deps; fix eslint issues; fix bug with collections

transitive-bullshit committed:
Merge pull request #707 from transitive-bullshit/feature/upkeep-june-2025

transitive-bullshit committed:
fix: update react-notion-x deps

transitive-bullshit committed:
feat: add maxDepth to limit depth of pages in space to pre-render

transitive-bullshit committed:
feat: improve env parsing

transitive-bullshit committed:
fix: REDIS_ENABLED issue

onnimonni committed:
Add JSON-LD microdata for blog posts

transitive-bullshit committed:
Merge pull request #653 from onnimonni/add-json-ld-microdata-for-blog-posts

Armruo committed:
fix: resolve ReferenceError for undefined isBlogPost in PageHead component

- Add isBlogPost as optional boolean prop to PageHead component
- Pass isBlogPost prop from NotionPage to PageHead component
- Fixes runtime error when rendering blog post structured data

transitive-bullshit committed:
Merge pull request #722 from Armruo/fix/pagehead-isblogpost-undefined

transitive-bullshit committed:
fix: collection / database loading

transitive-bullshit committed:
feat: bump core deps

transitive-bullshit committed:
fix: upgrade notion-client to fix collection issues

wustep committed:
fix: include pageUrlOverrides in getStaticPaths for production builds

URL override paths (e.g., /articles, /notes) may not be in the sitemap
if they're not directly linked from the root Notion page. This ensures
they are pre-rendered at build time, fixing 404s in production.

transitive-bullshit committed:
Merge pull request #742 from wustep/fix/static-paths-url-overrides

transitive-bullshit committed:
🖐

transitive-bullshit committed:
feat: update core deps

transitive-bullshit committed:
🚷

transitive-bullshit committed:
🥂

transitive-bullshit committed:
⛑

transitive-bullshit committed:
Merge pull request #748 from transitive-bullshit/feature/update-feb-2026

feat: update deps and next.js

transitive-bullshit committed:
🍿

transitive-bullshit committed:
🗣

transitive-bullshit committed:
⌛

transitive-bullshit committed:
🔣

transitive-bullshit committed:
😝

transitive-bullshit committed:
feat: remove react-icons in favor of local icons

transitive-bullshit committed:
feat: update react-notion-x

transitive-bullshit committed:
👏

transitive-bullshit committed:
🏮

transitive-bullshit committed:
⏭
